docs/source/reference/auth.rst

```diff
@@ -12,38 +12,114 @@ SkyPilot API server supports two authentication methods:
 .. N+1: Proxy authentication, where the reverse proxy in front of the API server handles the authentication and pass the identity header to the API server. This is mutually exclusive with authentication schemes 1~N. For clarity maybe this part will be hosted in another doc.
 .. TODO(aylei): replace basic auth with proxy auth for clarity after we support service account token based authentication to be used along.
 
-- **Basic auth**: Use an admin-configured username and password to authenticate.
+- **Basic auth in ingress**: Use an admin-configured username and password to authenticate.
+- **Token-based basic auth**: Use a token to authenticate.
```

Comment:
How about "Service account token"? Looks like we are not doing basic auth here

Reply:
Updated